code/API_definitions/check-device-connectivity.yaml

```diff
@@ -79,6 +79,46 @@ paths:
           $ref: "#/components/responses/Generic500"
         '503':
           $ref: "#/components/responses/Generic503"
+   /connectionStatus:
+    post:
+      tags:
+        - Check Device Connection
+      summary: 'Check if the device currently is connected to the network'
+      operationId: checkConnectionStatus
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RequestConnectionStatus'
+        required: true
+      responses:
+        '200':
+          description: Contains information about currently connection status to the network
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NetworkConnectionEventReport'
+              examples:
+                No-Country-Code:
+                  value:
+                    - ueId:
+                        ipv6Addr: 2001:db8:85a3:8d3:1319:8a2e:370:7344
+                      isConnected: true
+                    - ueId:
+                        msisdn: '123456789'
+                      isConnected: false
```

Comment:
Rename the example to ```Multiple-Devices``` or something like that

Reply:
Or you remove the whole "examples", because it's just needed when you have multiple different examples.

Reply:
I renamed it. It should be deleted?